pkg/providers/tinkerbell/hardware/csvparser.go

```diff
@@ -6,13 +6,18 @@ import (
 	"os"
 )
 
+const (
+	HeaderNotExist = -1
+)
+
```

Comment:
nit: Since we have a single constant.

```suggestion
const HeaderNotExist = -1
```